tests/framework/validators/UniqueValidatorTest.php

```
@@ -536,6 +536,30 @@ public function testSecondTargetAttributeWithError(): void
+        ]);
+        $model = new OrderItem();
+        $model->order_id = 1;
+        $model->item_id = 1;
+        $val->validateAttribute($model, 'order_id');
+        $this->assertTrue($model->hasErrors('order_id'));
+    }
+
+    public function testApplyTableAliasWithQuotedColumn(): void
+    {
+        $val = new UniqueValidator();
+        $method = new \ReflectionMethod($val, 'applyTableAlias');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '555', column '23'). (undefined)

(MissingImport)